docs/MAIL-ETIQUETTE

```diff
@@ -200,10 +200,10 @@ MAIL ETIQUETTE
 
   This is why top posting is so bad:
 
-      A: Because it messes up the order in which people normally read text.
       Q: Why is top-posting such a bad thing?
-      A: Top-posting.
+      A: Because it messes up the order in which people normally read text.
       Q: What is the most annoying thing in e-mail?
+      A: Top-posting.
```

Comment:
I see, I did not get that ;).
Changed to:  
This is why top posting is so bad (in top posting order):

      A: Because it messes up the order in which people normally read text.
      Q: Why is top-posting such a bad thing?
      A: Top-posting.
      Q: What is the most annoying thing in e-mail?